src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -1562,7 +1576,16 @@ private void parsePageNumber(
         String refId = getAttr(event, "ref-id");
         NumeralStyle style = getNumeralStyle(event);
         scanEmptyElement(input, ObflQName.PAGE_NUMBER);
-        fc.insertReference(refId, style);
+        fc.insertPageReference(refId, style);
+    }
+
+    private void parseMarkerReference(
+        BlockContentBuilder fc,
+        XMLEvent event, XMLEventIterator input,
+        TextProperties tp
+    ) throws XMLStreamException {
+        fc.insertMarkerReference(parseMarkerReferenceField(event, false), tp);
+        scanEmptyElement(input, ObflQName.MARKER_REFERENCE);
```

Comment:
For symmetry reasons, please have the same order as in the previous method.
That is, first `scanEmptyElement` and then `fc.insert...`

Reply:
No remarks and no change?

Reply:
But it's not consistent within the file. And TBH I think it's really unimportant.